### PR TITLE
Backing out the separate keyhash change

### PR DESCRIPTION
**Pull request by JaredCorduan**

### CODE REVIEW

File: shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Serialization.hs [hunk 1]
```diff
@@ -246,12 +248,12 @@ checkEncoding ::
   TestTree
 checkEncoding encode decode name x t =
   testCase testName $
-    assertEqual testName (fromEncoding $ toCBOR t) (fromEncoding $ encode x)
+    assertEqual testName
+      (Base16.encode $ serialize t)
+      (Base16.encode . serializeEncoding . encode $ x)
```
Comment (JaredCorduan) [inline shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Serialization.hs:253]: this makes it _a lot_ easier to debug failures. you can now just dump the failure into cbor.me.

**Review by uroboros — APPROVED** [review]
Farewell `KeyRole` - gone too soon

**Review by dcoutts — APPROVED** [review]

**Comment by JaredCorduan** [comment]
The property tests have timed out on me three times now, I would like to disable them from the CI tests. They will still run in the nightly tests, though. And we can re-enable them once we get the times back to something reasonable.

**Comment by uroboros** [comment]
> The property tests have timed out on me three times now, I would like to disable them from the CI tests. They will still run in the nightly tests, though. And we can re-enable them once we get the times back to something reasonable.

I'm chasing an UnexpectedDepositPot STS fail that is causing the property tests to take more time - it has to do with catching up the generators with recent changes to the exec spec - I'll bring the property tests back asap

**Pull request merged**